install/froxlor.sql

```diff
@@ -655,7 +655,8 @@ INSERT INTO `panel_languages` (`id`, `language`, `iso`, `file`) VALUES
     (4, 'Portugu&ecirc;s', 'pt', 'lng/portugues.lng.php'),
     (5, 'Italiano', 'it', 'lng/italian.lng.php'),
     (6, 'Nederlands', 'nl', 'lng/dutch.lng.php'),
-    (7, 'Svenska', 'sv', 'lng/swedish.lng.php');
+    (7, 'Svenska', 'sv', 'lng/swedish.lng.php'),
+    (8, 'Polish', 'pl', 'lng/polish.lng.php');
```

Comment:
Hi i remember this sorry 
i check this I lost some little lines
But We make this that (For backup)
Give me 1 day i should give final commit